### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mikeweiwei**

### CODE REVIEW

File: def/configmap/main.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ type options struct {
 	output    string
 }
 
-// multiKeyValue allows --key=value --key=value
+// multiKeyValue allows --key=value
```
Comment (justinsb) [inline def/configmap/main.go]: I think changing this to `--key=value --key=value2` would better capture the intent of the comment, I believe

File: experiment/bootstrap/repos.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ func (r *Repo) GitBasePath(ssh bool) string {
 
 // PullNumbers converts a Pull's list string into a slice of PR number strings
 // NOTE: this assumes that if there are pull requests specified that Repo.Pull
-// looks something like: `master:hash,prNum:hash,prNum:hash,...`
+// looks something like: `master:hash,prNum:hash,...`
```
Comment (justinsb) [inline experiment/bootstrap/repos.go]: Looking at the code, I think the comment is correct (?)... note that it returns a `[]string`

File: prow/plugins/approve/approvers/owners.go [hunk 3]
```diff
@@ -447,7 +447,7 @@ func (ap Approvers) GetFiles(org, project, branch string) []File {
 // remove those that are not useful considering current approvers and
 // assignees. This only uses leave approvers to find approvers the
 // closest to the changes.
-// - We find a subset of suggested approvers from from current
+// - We find a subset of suggested approvers from current
```
Comment (justinsb) [inline prow/plugins/approve/approvers/owners.go]: This one LGTM :-)

**Comment by justinsb** [comment]
I propose tweaking your 1st change, leaving the 2nd comment as is (not changing it), and the 3rd change looks good.

**Comment by mikeweiwei** [comment]
> 
> 
> I propose tweaking your 1st change, leaving the 2nd comment as is (not changing it), and the 3rd change looks good.

ok~~~

**Pull request merged**